src/teller/teller.go

```diff
@@ -71,6 +81,24 @@ func (s *Teller) Shutdown() {
 	<-s.done
 }
 
+// Backup starts a timestamped backup download
+func (s *Teller) Backup() http.HandlerFunc {
```

Comment:
Why not pass the db to Monitor and add the handler there?

Reply:
More semantic?
Like saying taking backup for teller

Reply:
seems like indirection, only monitor would ever use it, 

teller.Teller exposes the public bind/status API, backup is not a function of the client interface